change/@ni-nimble-components-46e491a1-b49a-4d5c-b64e-6eb6fae50e0b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@Razvan1928 one thing I want to mention is that it does seem like your current workflow is to push each individual commit remote at a time. I just want to point out that each commit pushed remote does a PR build so this PR as an example has run over 100 builds.

It is different from Skyline where you have to explicit queue a build. It is not necessarily a problem and if it useful it is fine. But if you felt that you could create multiple commits locally and push them together so fewer builds are created and that does not impact your workflow that could be a small tweak to reduce the number of builds

![image](https://github.com/ni/nimble/assets/1588923/495c2765-c409-4ceb-ad93-428b6267f68f)